src/Text/Text.tsx

```diff
@@ -222,8 +334,6 @@ const Container = styled(Box)<IText>(
       letter-spacing: 0.75px;
     `}
 
-    margin: 0;
-    padding: 0;
```

Comment:
Isn't this gonna create some issue on some repos? Did you check repos reset files to be sure that everything is properly setup?

Reply:
yeah all repos are resetting margin and padding in global style and these two were breaking the new margin props